Jenkinsfile

```diff
@@ -55,24 +55,23 @@ pipeline {
 
             steps {
               sh 'mkdir -p .npm'
-              sh """
-              npm install
-              """
+              sh 'npm install'
             }
           }
 
           stage('Test') {
             steps {
-              sh 'node -v'
-              sh 'npm -v'
-              sh 'npm test'
+              sh 'mkdir -p coverage'
+              sh 'npm run test'
```

Comment:
this should run test:ci so the shell scripts runs and it doesn't need to be baked into the jenkins file

Reply:
This change was backed out.  Don't need the bash script any more.